include/NovelRT/Maths/Utilities.h

```diff
@@ -22,17 +22,30 @@ namespace NovelRT::Maths
     class Utilities
     {
     public:
+        /**
+         * @brief Returns the de Bruijn Sequences to Index a 1 in a Word
+         * @details To learn more about the de Bruijn Sequences, refer to this documentation: http://supertech.csail.mit.edu/papers/debruijn.pdf
+         * @returns A 32 items array of 8 bits integer containing the de Bruijn Sequences
+         */
         [[nodiscard]] inline constexpr static std::array<uint8_t, 32> Log2DeBruijn() noexcept
         {
             return std::array<uint8_t, 32>{0, 9,  1,  10, 13, 21, 2,  29, 11, 14, 16, 18, 22, 25, 3, 30,
                                            8, 12, 20, 28, 15, 17, 24, 7,  19, 27, 23, 6,  26, 5,  4, 31};
         }
 
+        /**
+         * @brief Check if a given integer is an integral power of two.
+         * @return Returns true if a given integer is an integral power of two, otherwise false
+         */
```

Comment:
Only minor stuff here!
```suggestion
        /**
         * @brief Check if a given integer is an integral power of two.
         * @return true if a given integer is an integral power of two, otherwise false.
         */
```